packages/instantsearch.js/src/widgets/hits/defaultTemplates.ts

```diff
@@ -1,14 +1,14 @@
 import { omit } from '../../lib/utils';
 
-import type { HitsTemplates } from './hits';
+import { HitsTemplates } from './hits';
```

Comment:
Why?

Reply:
eslint complains that it's needed for "decorator metadata", probably as a false positive. Will ignore the rule instead indeed